firmware/application/ui/ui_rssi.cpp

```diff
@@ -75,8 +75,9 @@ void RSSI::paint(Painter& painter) {
 		Color::black()
 	);
 	
-	if (pitch_rssi_enabled)
+	if (pitch_rssi_enabled) {
```

Comment:
This small changes should be included in another PR, or part of a "format cleanup". It is fine for now, but just keep in mind that a cleaner PR is always better :)